java/src/org/openqa/selenium/remote/AbstractDriverOptions.java

```diff
@@ -94,4 +135,15 @@ public Object getCapability(String capabilityName) {
     getExtraCapabilityNames().forEach(name -> toReturn.put(name, getCapability(name)));
     return Collections.unmodifiableMap(toReturn);
   }
+
+  private Map<String, Number> getTimeouts() {
+    Map<String, Number> newTimeouts = new HashMap<>();
+    Object raw = getCapability(TIMEOUTS);
```

Comment:
You need to check that `raw` is actually a `Map` before casting to it. If not, if the user has set some garbage value (eg: `setCapability(TIMEOUTS, "cheese")` chaos will follow.